tests/HolidaysTest.php

```diff
@@ -91,3 +91,9 @@
     $result = Holidays::has(country: 'unknown');
     expect($result)->toBeFalse();
 });
+
+it('can get translated holiday names', function () {
+    $result = Holidays::for(country: 'be', year: 2020, locale: 'fr')->get();
+
+    expect(formatDates($result))->toMatchSnapshot();
```

Comment:
We don't have to verify everything in the result, only that we get a translated name. I would use a specific assertion instead of a snapshot one